src/port.js

```diff
@@ -1,7 +1,19 @@
 class Port {
-    constructor(name) {
+    constructor (name) {
         this.name = name;
+        this.ships = [];
     }
-}
 
-module.exports = Port;
\ No newline at end of file
+    addShip(ship) {
+        this.ships.push(ship);
+    }
+
+    removeShip(ship) {
+        const indexOfShip = (this.ships.indexOf(ship));
+        this.ships.splice(indexOfShip);
```

Comment:
when using splice, the second argument it takes is a number that tells you how many items should be removed from that array starting from the first argument. Because you've left it out, it will remove all ships from the port after the index of the ship you want to remove, you should instead replace this with `this.ships.splice(indexOfShip, 1)` to only remove the one ship

Reply:
Also, you can remove the parenthesis around `(this.ship.indexOf(ship))`